doc/classes/TreeItem.xml

```diff
@@ -568,14 +568,14 @@
 				Sets the given column's custom color.
 			</description>
 		</method>
-		<method name="set_custom_draw" deprecated="Use [method TreeItem.set_custom_draw_callback] instead.">
+		<method name="set_custom_draw" deprecated="This method was used in Godot 3.x. Use [method TreeItem.set_custom_draw_callback] instead.">
```

Comment:
```suggestion
		<method name="set_custom_draw" deprecated="Use [method TreeItem.set_custom_draw_callback] instead.">
```
It is also in 4.0, 4.1, and 4.2, it was added just a few weeks ago, so this is misleading IMO